src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -477,7 +477,7 @@ private function getLinks(ResourceMetadataCollection $resourceMetadataCollection
                     $operationName === $operation->getName() ||
                     isset($links[$operationName]) ||
                     $operation instanceof CollectionOperationInterface ||
-                    HttpOperation::METHOD_GET !== ($method ?? null)
```

Comment:
Same: not sure about this change, should it be rollbacked too?

Reply:
same an httpoperation always has a method